### PR TITLE
Guess-the-Sketch: Move to embeddings API

### PR DESCRIPTION
**Pull request by zmerlynn**
Steals code from https://github.com/googleforgames/GenAI-quickstart/blob/main/genai/language/embeddings/src/example_api_call.py

Also round the answer in the front-end

### CODE REVIEW

**Review by gongmax — APPROVED** [review]

File: examples/guess-the-sketch/src/app.py [hunk 1]
```diff
@@ -213,5 +209,21 @@ def handle_message(data):
             # Send the first picture to the opponent for guessing
             emit('guess_sketch_response', {'image': player_prompt[player][1]['picture'], 'prompt': player_prompt[player][1]['prompt'], 'opponentId': player, 'round': 1}, room=player_id)
 
+def similarity(p1, p2):
+    '''Calculate dot product distance between two prompts'''
+
+    resp = requests.post(
```
Comment (roberthbailey) [inline examples/guess-the-sketch/src/app.py:215]: Does this need error handling? What if the embeddings endpoint is down?

Comment (zmerlynn) [reply examples/guess-the-sketch/src/app.py:215]: line 220 :)

Comment (zmerlynn) [reply examples/guess-the-sketch/src/app.py:215]: `raise_for_status` is just a convenience function that says "raise an Exception if it's anything other than `200`.

Comment (roberthbailey) [reply examples/guess-the-sketch/src/app.py:215]: I'm wondering when the exception gets raised what happens? It bubbles up to the calling function which .... crashes? Does anything come back to the UI?

**Review by roberthbailey — APPROVED** [review]

**Pull request merged**